examples/plotting/plotting_blank_map.py

```diff
@@ -53,12 +53,35 @@
 xc = [0, 50, 400] * u.arcsec
 yc = [0, 100, 400] * u.arcsec
 
+################################################################################
+# Note that the blue dot corresponding to (0, 0) in helioprojective coordinates is
+# not at the intersection of the heliographic equator and the heliographic prime
+# meridian. (0, 0) in helioprojective coordinates *is* at the center of the solar
+# disk, and at the center of the overall image. The reason that the intersection of
+# the two heliographic grid lines is not also at disk center is because the observer
+# (specified as Earth in this example) is at non-zero heliographic latitude (because
+# the Sun's equatorial plane is tilted relative to the ecliptic plane that contains
+# Earth's orbit). Disk center as seen by an observer has the same heliographic
+# latitude as the heliographic latitude of the observer, and this value is known as
+# the B0 angle, and varies over the year.
+#
+# To have no no offset (i.e., the heliographic equator is a horizontal line), you
+# can specify an observer on the Sun's equatorial plane instead of using Earth as
+# the observer.
+
+# sphinx_gallery_defer_figures
+
+observer_sun = SkyCoord(0*u.deg, 0*u.deg, 1*u.AU, obstime='2013-10-28',
+                    frame=frames.HeliographicStonyhurst)
+skycoord_sun = SkyCoord(0*u.arcsec, 0*u.arcsec, obstime='2013-10-28',
+                    observer=observer_sun, frame=frames.Helioprojective)
+
```

Comment:
```suggestion
```
What you are doing here could have value, but is completely unsupported by the text.  The sentence above – which I want to remove anyway – is about creating a map with a horizontal-looking equator.  Plotting disk center as seen by a different observer is not the same thing at all.